package.json

```diff
@@ -26,16 +26,18 @@
   "homepage": "https://github.com/RangerMauve/random-access-web#readme",
   "dependencies": {
     "@sammacbeth/random-access-idb-mutable-file": "^0.1.1",
-    "random-access-chrome-file": "^1.1.2",
+    "random-access-chrome-file": "^1.1.4",
     "random-access-idb": "^1.2.1",
     "random-access-idb-mutable-file": "^0.3.0",
     "random-access-memory": "^3.1.1",
-    "random-access-storage": "^1.3.0"
+    "random-access-storage": "^1.4.1",
+    "@DougAnderson444/random-access-idb": "github:DougAnderson444/random-access-idb"
```

Comment:
What are the changes here used for?

Reply:
That's to address my comment [here](https://github.com/random-access-storage/random-access-web/issues/6#issuecomment-702969661). 

I merged the substack changes into random-access-idb since neither seemed to work on their own, but together they made magic happen. 

Put this here until the `r-a-idb` can be pulled in.

Reply:
Looks like my PR on r-a-idb will soon be merged though, so this could become a non-issue
https://github.com/random-access-storage/random-access-idb/pull/9#partial-pull-merging